tools/csv-checksum/csv-checksum.go

```diff
@@ -114,6 +115,7 @@ func main() {
 	md5Hash = addArgToHash("--ceph-image", *cephContainerImage, md5Hash)
 	md5Hash = addArgToHash("--rook-csi-ceph-image", *rookCsiCephImage, md5Hash)
 	md5Hash = addArgToHash("--rook-csi-registrar-image", *rookCsiRegistrarImage, md5Hash)
+	md5Hash = addArgToHash("--rook-csi-resizer-image", *rookCsiRegistrarImage, md5Hash)
```

Comment:
```suggestion
	md5Hash = addArgToHash("--rook-csi-resizer-image", *rookCsiResizerImage, md5Hash)
```

Reply:
I have also noticed that, it should be fixed, now.

Reply:
I still don't see the changes pushed.

Reply:
Ah, yes, it turned out I forgot to ammend the commit before pushing. :-/